Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM fedora:20
+FROM centos:7
 MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 
 # Execute system update
-RUN yum -y update && yum clean all
+RUN yum -y update && rm -rf /var/cache/yum
```

Comment:
Hey, just asking! :) It should probably be safe, although many people look at the base image source and I guess I would better see that people use `yum` instead `rm` in their Dockerfiles :P

I would probably stick with `yum`, especially after looking at [this](http://yum.baseurl.org/gitweb?p=yum.git;a=blob;f=cli.py;h=fd6e7159c450f4c8726b5b50a06e666dc3665a12;hb=HEAD#l1657).